api/db/changelog/db.changelog-118-update-demographics.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="srubenst" id="db.changelog-116-update-demographics">
+    <addColumn tableName="user">
+      <column name="professional_url" type="varchar(255)">
+        <constraints nullable="true"/>
+      </column>
+    </addColumn>
+    <addColumn tableName="demographic_survey">
+      <column name="identifies_as_lgbtq" type="boolean">
+        <constraints nullable="true"/>
+      </column>
+      <column name="lgbtq_identity" type="varchar(255)">
```

Comment:
[Just thinking out loud] Have we encountered scenarios where users attempt to put more text into an input form than our database column width allows? Do we have clear DB design guidelines around choosing between the various text types or lengths for free-form text like this? I suspect 255 is probably enough here, I'm just curious if there is a broader pattern.

(Based on a quick dig into our codebase, it seems we use varchar more often by far, with one instance of type="longtext" and a few instances of type="text" in the data dictionary schema. It's a clear pattern, but I'm not sure the motivation behind preferring varchar more often than not.)

Reply:
We haven't. I can add some front-end validation on that though. I asked Lou and it sounded like we thought 255 char was probably enough for this. I've been mainly following the DB schema pattern we have established.